Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -0,0 +1,76 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import WebKit
+
+// TODO: RUMM-1794 rename the method
+public extension WKUserContentController {
+    func addDatadogMessageHandler(allowedWebViewHosts: [String]) {
```

Comment:
How about unit testing this by subclassing `WKUserContentController` and spying calls to `add(messageHandler, name: bridgeName)` and `addUserScript()`? This way we will test basic, but fundamental aspect of the bridge configuration.